doc/win-dev.ps1

```diff
@@ -70,25 +70,25 @@ try {
 	$Env:Path += $ChocoPath
 }
 
-# GitHub Actions uses an image that comes with most dependencies preinstalled. Don't install them twice.
+choco install -y `
+    "visualstudio${VsVersion}community" `
+    "visualstudio${VsVersion}-workload-netcoretools" `
+    "visualstudio${VsVersion}-workload-vctools" `
+    "visualstudio${VsVersion}-workload-manageddesktop" `
+    "visualstudio${VsVersion}-workload-nativedesktop" `
+    "visualstudio${VsVersion}-workload-universal" `
+    "visualstudio${VsVersion}buildtools" `
```

Comment:
Now as the GHA installs much more stuff, I myself would rather:

* Do nothing
* or #9747 (but that's definitely something for a .0 version)